plugin/pkg/admission/nodetaint/admission.go

```diff
@@ -27,30 +28,40 @@ import (
 
 const (
 	// PluginName is the name of the plugin.
-	PluginName = "TaintNodesByCondition"
```

Comment:
changing an existing admission plugin's name is not backwards compatible

Reply:
> a webhook admission plugin could be used to do this, rather than adding config to an in-tree admission plugin

@liggitt Thank you for you advice. I'm not sure if others also encounter the same problem. If the answer is yes , maybe it's better to make a general solution in admission plugin.